index.html

```diff
@@ -15,159 +15,204 @@
+                <li class="top-nav-items-desktop">Portfolio</li>
+                <li class="top-nav-items-desktop">About</li>
+                <li class="top-nav-items-desktop">Contact</li>
+            </ul>
         </nav>
     </header>
-    <section class="headline-section">
-        <h1 class="heading-1">I'm Abdul <br> Glad to see you!</h1>
-        <p class="intro-info">I’m a software developer! I can help you build a product , feature or website Look
-            through some of my work and experience! If you like what you see and have a project you need coded,
-            don’t hestiate to contact me.</p>
-        <h2 class="connect-info">LET’S CONNECT</h2>
-
-        <ul class="social-icons">
-            <li><a href="twitter.com"><img class="logo-images" src="./assets/icons/Twitter_logo.svg"
-                        alt="Twitter_logo"></a></li>
-            <li><a href="linkedin.com"><img class="logo-images" src="./assets/icons/Linkedin_icon.svg"
-                        alt="Linkedin Icon"></a></li>
-            <li><a href="medium.com"><img class="logo-images" src="./assets/icons/Medium.svg" alt="Medium logo"></a>
-            </li>
-            <li><a href="Github.com"><img class="logo-images" src="./assets/icons/Github_icon.svg"
-                        alt="Github logo"></a></li>
-            <li><a href="angelist.com"><img class="logo-images" src="./assets/icons/Angelist.svg"
-                        alt="Angelist logo"></a></li>
-        </ul>
-    </section>
-    <main>
+    <main class="main-part">
+        <section class="headline-section">
+            <h1 class="heading-1">I'm Abdul <br> Glad to see you!</h1>
+            <p class="intro-info">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <h2 class="connect-info">LET’S CONNECT</h2>
+
+            <ul class="social-icons">
+                <li><a href="twitter.com"><img class="logo-images" src="./assets/icons/Twitter_logo.svg"
+                            alt="Twitter_logo"></a></li>
+                <li><a href="linkedin.com"><img class="logo-images" src="./assets/icons/Linkedin_icon.svg"
+                            alt="Linkedin Icon"></a></li>
+                <li><a href="medium.com"><img class="logo-images" src="./assets/icons/Medium.svg" alt="Medium logo"></a>
+                </li>
+                <li><a href="Github.com"><img class="logo-images" src="./assets/icons/Github_icon.svg"
+                            alt="Github logo"></a></li>
+                <li><a href="angelist.com"><img class="logo-images" src="./assets/icons/Angelist.svg"
+                            alt="Angelist logo"></a></li>
+            </ul>
+        </section>
+    
         <!-- First Page -->
         <section class="work">
             <article class="work-section">
```

Comment:
- [ ] As the previous reviewer suggested your current card design is not following the Figma template. Kindly fix it, so your design can be as close to the Figma design as possible

|Your design|Figma design|
|-|-|
|![Screenshot_20220603_200024](https://user-images.githubusercontent.com/25227022/171931501-62d5c841-f9fd-42cf-a49e-cc0b0865228e.png)|![Screenshot_20220603_200516](https://user-images.githubusercontent.com/25227022/171931642-091a8e56-fe45-4461-a15a-6457defad946.png)|

Reply:
Hello Dear reviewer,

The issue with the last code review was solved. As the previous reviewer suggested that I fix the overall width of the card so that it doesn't exceed the width when displayed in bigger screens. I contacted the previous code reviewer and asked him to clarify what the issue was and he did. I totally fixed the issue if you put the website into a bigger display exceeding my display the card won't become bigger in width. I think I have solved the issue stated by the previous reviewer. Can you please check what the required changes of the previous reviewer were and then take a look at my project please. Since we all have limited code reviews and I believe I have solved the issue which was stated by the previous reviewer.

Regards